seqr/views/apis/staff_api_tests.py

```diff
@@ -195,3 +202,75 @@ def test_elasticsearch_status(self, mock_elasticsearch, mock_get_mapping):
                                                       h="index,docs.count,store.size,creation.date.string")
         mock_es_client.cat.aliases.assert_called_with(format="json", h="alias,index")
         mock_get_mapping.assert_called_with(doc_type='variant,structural_variant')
+
+    def test_mme_details(self):
+        url = reverse(mme_details)
+        _check_login(self, url)
+
+        response = self.client.get(url)
+        self.assertEqual(response.status_code, 200)
+        response_json = response.json()
+        self.assertListEqual(response_json.keys(), ['metrics', 'genesById', 'submissions'])
+
+    def test_seqr_stats(self):
+        url = reverse(seqr_stats)
+        _check_login(self, url)
+
+        response = self.client.get(url)
+        self.assertEqual(response.status_code, 200)
+        response_json = response.json()
+        self.assertListEqual(response_json.keys(), ['individualCount', 'familyCount', 'sampleCountByType'])
```

Comment:
As I said above, please test the content

Reply:
I could add tests against static values. But it could failed when the fixture data were changed. Or should I query the database to get the expected results? I do the test statically for now.